src/main/java/wooteco/subway/path/application/PathService.java

```diff
@@ -0,0 +1,37 @@
+package wooteco.subway.path.application;
+
+import org.springframework.stereotype.Service;
+import wooteco.subway.line.dao.SectionDao;
+import wooteco.subway.line.domain.Sections;
+import wooteco.subway.path.domain.Path;
+import wooteco.subway.path.dto.PathResponse;
+import wooteco.subway.station.dao.StationDao;
+import wooteco.subway.station.domain.Station;
+import wooteco.subway.station.dto.StationResponse;
+
+import java.util.List;
+
+@Service
+public class PathService {
+
+    SectionDao sectionDao;
+    StationDao stationDao;
```

Comment:
```suggestion
    private SectionDao sectionDao;
    private StationDao stationDao;
```

접근제어자를 생략한 것은 의도된 내용일까요!?